.github/workflows/benchmark.yml

```diff
@@ -1,8 +1,10 @@
 name: Benchmark
 
-# only one can tun at a time.
+# only one can run at a time.
 # Actions access a common cache entry and may corrupt it.
-concurrency: cd-benchmark-${{ github.ref }}
```

Comment:
Note this change breaks the original intent of running strictly once per github ref

Reply:
you mean strictly one at a time?

if i change it to : `${{ github.workflow }}-${{ github.ref }}` would it be ok to cancel the previous ongoing run?

or I can copy the benchmark which runs in original way on stable, unstable and benchmark , and modify this to run on the PRs only with the cancel-in-progess true